backend/geonature/core/imports/routes/__init__.py

```diff
@@ -12,3 +14,15 @@ def list_destinations():
     destinations = Destination.query.all()
     # FIXME: filter with C permissions?
     return schema.dump(destinations, many=True)
+
+
+@blueprint.route("/destination/<destinationCode>", methods=["GET"])
+@login_required
+def get_destination(destinationCode):
+    schema = DestinationSchema(only=["module"])
+    destination = db.session.execute(
+        db.select(Destination)
+        .options(joinedload("module"))
+        .where(Destination.code == destinationCode)
```

Comment:
**Suggestion:** In the `get_destination` function, you are using `db.session.execute` to execute a raw SQL query. This is not recommended as it can lead to SQL injection vulnerabilities if not handled properly. Instead, you should use SQLAlchemy's ORM query methods which are safer and more readable. [security]
```suggestion
@blueprint.route("/destination/<destinationCode>", methods=["GET"])
@login_required
def get_destination(destinationCode):
    schema = DestinationSchema(only=["module"])
    destination = Destination.query.options(joinedload("module")).filter_by(code=destinationCode).first()
    return schema.dump(destination)
```